banyand/kv/kv.go

```diff
@@ -160,6 +171,9 @@ func OpenTimeSeriesStore(shardID int, path string, options ...TimeSeriesOptions)
 	}
 	// Put all values into LSM
 	btss.dbOpts = btss.dbOpts.WithVLogPercentile(1.0)
+	if btss.dbOpts.MemTableSize < 8<<20 {
```

Comment:
`8<<20` seems to be a magic number... What does it mean?

Reply:
8MB